compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java

```diff
@@ -4024,6 +4030,24 @@ boolean validateFloatLiteral(Location pos, String numericLiteral) {
         return true;
     }
 
+    boolean validateDecimalLiteral(Location pos, String decimalLiteral) {
```

Comment:
Shall we rename the name the method to `isValidDecimalNumber` or similar, instead of mentioning literal? because I can use this function in my fix #37726 as well

Reply:
Changed